src/component/shortcuts.js

```diff
@@ -77,8 +77,13 @@ export default class extends React.Component {
       let isInputLikeElement = element.tagName === 'INPUT' ||
         element.tagName === 'SELECT' || element.tagName === 'TEXTAREA' ||
           (element.contentEditable && element.contentEditable === 'true')
-      let isReturnString = event.key && event.key.length === 1
-
+        var isReturnString;
```

Comment:
Sorry, I edited a compiled version of the file during my tests and I did see that the original file had that crazy codestyle too. Will do.